source/whats-new.txt

```diff
@@ -0,0 +1,61 @@
+.. _entity-framework-whats-new:
+
+==========
+What's New
+==========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: update, new feature, deprecation, upgrade
+
+Learn what's new in:
+
+* :ref:`Version 8.1 <entity-framework-version-8.1>`
+
+.. important::
+
+   To ensure that the provider major version number aligns with
+   {+framework-core+} versions, the {+provider-long+} does not limit breaking
+   changes to major version releases. Instead, breaking and significant changes
+   are made in minor version releases.
+ 
+.. _entity-framework-version-8.1:
+
+What's New in 8.1
+-----------------
+
+.. warning:: Breaking Changes
+
+   The v8.1 provider introduces the following breaking changes:
+
+   - The {+provider-short+} runs all changes as MongoDB transactions. If your
+     application uses a {+mdb-server+} version earlier than v5.0, or does not run in a
+     replica set, you must reconfigure your application.
+   - Changed the default GUID storage format from ``CSharpLegacy`` binary
+     format to ``Standard`` format.
+   - Modified the ``IMongoClientWrapper`` interface to provide service
+     registration for the ``MongoClientWrapper`` class. You should not implement the
+     ``IMongoClientWrapper`` interface in your application.
+   - The ``CamelCaseElementNameConvention`` cases owned entities based on their
+     property name instead of their class name.
+
+   For more information about these breaking changes, see
+   :ref:`entity-framework-breaking-changes-v8.1` in the Upgrade Provider Version guide.
+
+The 8.1 provider release includes the following new features:
+
+- Support for optimistic concurrency through the
+  ``IsConcurrencyToken()`` method or the ``ConcurrencyCheck()`` attribute
```

Comment:
Recommend replacing with:

-   ``IsConcurrencyToken()`` method or the ``ConcurrencyCheck`` attribute 
-   ``IsRowVersion()`` method or the ``Timestamp`` attribute